### PR TITLE
Add filters to streams

### PR DESCRIPTION
**Pull request by aoudiamoncef**
@damip Please, If you could validated this changes

I'll not merge in main before finishing developing this feature.

### CODE REVIEW

File: proto/apis/massa/api/v1/public.proto [hunk 1]
```diff
@@ -669,7 +685,23 @@ message NewBlocksResponse {
...
+message NewBlocksHeadersRequest {
+  // Returns all the block headers that verify all the filters
+  repeated NewBlockHeadersBlocksFilter filters = 1;
+}
+
+// NewBlockHeadersBlocks Filter
+message NewBlockHeadersBlocksFilter {
+  // Filter
+  oneof filter {
+    // One of this block ids
```
Comment (damip) [inline proto/apis/massa/api/v1/public.proto:697]: this => these

File: proto/apis/massa/api/v1/public.proto [hunk 2]
```diff
@@ -660,7 +660,23 @@ message ScOutputEventsWrapper {
...
+message NewBlocksRequest {
+  // Returns all the blocks that verify all the filters
+  repeated NewBlocksFilter filters = 1;
+}
+
+// NewBlocks Filter
+message NewBlocksFilter {
+  // Filter
+  oneof filter {
+    // One of this block ids
```
[the remark this answers is not in the document]
Comment (damip) [reply proto/apis/massa/api/v1/public.proto:672]: elsewhere as well

**Review by damip — APPROVED** [review]
looks good to me :)

**Comment by damip** [comment]
Do you think we can have ti for testnet 26 ?

**Comment by aoudiamoncef** [comment]
I'll do It for tesntet 26

**Pull request merged**